core/templates/dev/head/pages/exploration-player-page/suggestion-modal-for-learner-local-view/suggestion-modal-for-exploration-player.service.ts

```diff
@@ -64,6 +64,11 @@ oppia.factory('SuggestionModalForExplorationPlayerService', [
             $scope.cancelSuggestion = function() {
               SuggestionModalService.cancelSuggestion($uibModalInstance);
             };
+
+            $scope.currentRteIsCKEditor4 = function() {
+              return CURRENT_RTE_IS_CKEDITOR_4;
+            };
```

Comment:
Do you need function here?

Reply:
Hi @vojtechjelinek, no it was not needed I've updated it Thanks!